Sources/ArcGISToolkit/Components/Form/TextInputFooter.swift

```diff
@@ -103,7 +103,7 @@ struct TextInputFooter: View {
         .foregroundColor(validationError == nil ? .secondary : .red)
         .onChange(of: text) { newText in
             if !hasPreviouslySatisfiedMinimum && !fieldType.isNumeric {
-                if newText.count >= lengthRange.lowerBound {
+                if newText.count >= lengthRange!.lowerBound {
```

Comment:
Is this OK if `lengthRange` is optional and can be nil?

Reply:
Should be. Length ranges are only applicable to non-numeric text input so the `!fieldType.isNumeric` condition above on 105 should protect there.